docs/gettingstarted.rst

```diff
@@ -1,16 +1,149 @@
+   virtualbox-guest-iso 5.1.22-1
+   virtualbox-host-modules-arch 5.1.22-2
+
+Start minikube
+--------------
+
+Once minikube is installed, you should start the Minikube VM:
+
+.. code-block:: console
+
+   $ minikube start
+
+and instruct your shell to use Minikube's Docker connection:
+
+.. code-block:: console
+
+   $ eval "$(minikube docker-env)"
+
+Install REANA
+-------------
+
+Install REANA server sources in a fresh Python virtual environment:
+
+.. code-block:: console
+
+   $ mkvirtualenv reana
+   $ pip install 'git+https://github.com/reanahub/reana.git@master#egg=reana'
+
+Initialise REANA cloud
+----------------------
+
+You can now initialise REANA cloud on the local minikube cluster:
+
+.. code-block:: console
+
+   $ reana init
+
+This will take several minutes to create the pods and download REANA compontent
+images. Once it finishes, let us check the REANA cloud pods:
+
+.. code-block:: console
+
+   $ kubectl get pods
+   job-controller-1390584237-0lt03        1/1       Running            0          1m
+   message-broker-1410199975-7c5v7        1/1       Running            0          1m
+   storage-admin                          1/1       Running            0          1m
+   workflow-controller-2689978795-5kzgt   1/1       Running            0          1m
+   workflow-monitor-1639319062-q7v8z      1/1       Running            0          1m
+   yadage-alice-worker-1624764635-x8z71   1/1       Running            0          1m
+   yadage-atlas-worker-2909073811-t9qv3   1/1       Running            0          1m
+   yadage-cms-worker-209120003-js5cv      1/1       Running            0          1m
+   yadage-lhcb-worker-4061719987-6gpbn    1/1       Running            0          1m
+   zeromq-msg-proxy-1617754619-68p7v      1/1       Running            0          1m
+
+Finally, we can test whether the REANA components are ready:
+
+.. code-block:: console
+
+   $ curl "http://$(reana get reana-workflow-controller)/workflows"
+   {
+     "workflows": {}
+   }
+   $ firefox "http://$(reana get reana-workflow-monitor)/helloworld"
+   $ curl "http://$(reana get reana-job-controller)/jobs"
+   {
+     "jobs": {}
+   }
+
+Run "hello world" example application
+-------------------------------------
+
+You can now submit a simple `"hello world" example application
+<https://github.com/reanahub/reana-demo-helloworld>`_ to run on your local REANA
+cloud:
+
+.. code-block:: console
+
+   $ reana run reanahub/reana-demo-helloworld -a atlas
+
+Let us verify whether it worked:
+
+.. code-block:: console
+
+   $ minikube ssh
+   minikube> find /reana/atlas/ -name greetings.txt -exec cat {} \;
+   minikube> Hello John Doe!
+
+Run "word population" example analysis
+--------------------------------------
+
+You can now submit a `"world population" example analysis
+<https://github.com/reanahub/reana-demo-worldpopulation>`_ to run on your local
+REANA cloud:
+
+.. code-block:: console
+
+   $ reana run reanahub/reana-demo-worldpopulation -a atlas
```

Comment:
Yup, typo.  But actually I'd rather remove it and make it optional. First, "experiment" is CERN-centric. Second, we don't support it well today anyway. So I'd amend `reana` script and make it optional as part of this PR.